ds/src/styles/tokens/index.css

```
@@ -1,11 +1,29 @@
+@import './_css/line-height.css';
+@import './_css/radius.css';
+@import './_css/shadow.css';
+@import './_css/spacing.css';
+@import './_css/z-index.css';
+
+@font-face {
+  font-style: normal;
+  font-weight: 100 900;
+  font-stretch: extra-condensed normal;
+  src: url('../../assets/fonts/geist-variable.woff2') format('woff2-variations');
+  font-family: 'Geist';
```

Stylelint (17.9.0)

[error] Expected no quotes around "Geist" (font-family-name-quotes)

(font-family-name-quotes)